File: Rnwood.Smtp4dev/ApiClient/AttachmentSummary.ts


export default class AttachmentSummary {

    constructor(fileName: string, contentId: string, url: string,) {

        this.fileName = fileName;
        this.contentId = contentId;
        this.url = url;
    }


    fileName: string;
    contentId: string;
    url: string;
}

File: Rnwood.Smtp4dev/ApiModel/AttachmentSummary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rnwood.Smtp4dev.ApiModel
{
    public class AttachmentSummary
    {
        public string FileName { get; set; }
        public string ContentId { get; set; }

		public string Id { get; set; }

		public string Url { get; set; }
    }
}
